kuwala/core/backend/app/resources/data_catalog_items.json

```diff
@@ -52,5 +52,36 @@
         }
       }
     ]
+  } , {
+    "id": "snowflake",
+    "name": "Snowflake",
+    "logo": "https://symbols.getvecta.com/stencil_96/10_snowflake-icon.4b623db584.svg",
```

Comment:
I've uploaded the SVG to our S3 bucket in case the URL changes.

Please use this URL for the logo: https://kuwala-public.s3.amazonaws.com/logos/snowflake-logo.svg

Reply:
Will replace it.

Reply:
![Screenshot from 2022-04-01 17-03-13](https://user-images.githubusercontent.com/45427673/161303196-e187466b-c0d4-481c-8fc1-41d49e45a145.png)

Added fields for **warehouse** and **role**. The **warehouse** field corresponds to a created **warehouse**  in the user's snowflake account. Also the **role** specifies a particular role that has privilege to the warehouse within the user's snowflake account.

![Screenshot from 2022-04-01 17-04-02](https://user-images.githubusercontent.com/45427673/161303268-98affeee-199d-4a5b-a197-a38e9f77c73d.png)

The corresponding profiles.yml file.